edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/IDeviceScopeApiClient.cs

```diff
@@ -2,9 +2,12 @@
 namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 {
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
 
     public interface IDeviceScopeApiClient
     {
+        IDeviceScopeApiClient CreateOnBehalfOfDeviceScopeClient(string deviceId);
```

Comment:
just want to confirm, the device id argument can be leaf device id or edge device id including both child and root. do I understand correctly?